File: src/main/java/cz/kamenitxan/df3d/Main.kt
package cz.kamenitxan.df3d

object Main {
	var port = 4567

	@JvmStatic
	fun main(args: Array<String>) {
		println("Chatlin Started")
		for (arg in args) {
			when {
				arg.contains("-port") -> port = arg.replace("-port=", "").toInt()
			}
		}

		val app = ChatlinApp().run()
	}
}

File: src/main/java/cz/kamenitxan/df3d/Main.scala
package cz.kamenitxan.df3d

import org.slf4j.LoggerFactory

object Main {
	private val logger = LoggerFactory.getLogger(this.getClass)

	def main(args: Array[String]) {
		val app =new DFS3DApp()
		app.run(args)
	}
}
